code/modules/research/designs/boards/computer_engie.dm

```diff
@@ -183,4 +183,14 @@
 	build_type = IMPRINTER
 	materials = list(MAT_GLASS = 2000, SACID = 20)
 	category = "Console Boards"
-	build_path = /obj/item/weapon/circuitboard/security/engineering
\ No newline at end of file
+	build_path = /obj/item/weapon/circuitboard/security/engineering
+
+/datum/design/fisson_reactor_controller
+	name = "Circuit Design (R-UST Mk. 7 core controller)"
+	desc = "Allows for the construction of circuit boards used to safely control a fission reactor."
+	id = "rust_core_control"
```

Comment:
Hmm, today I will review before the PR is complete. (it says R-UST when it shouldn't)